enzy_htp/structure/structure.py

```diff
@@ -1004,3 +1015,16 @@ def merge_right(left: Structure,
         if lkey not in right_keys:
             struct_cpy.add_residue(left.get_residue(lkey))
     return struct_cpy
+
+
+def order_to_stru(stru: Structure, ordered_stru: Structure) -> None:
+    """Orders the atoms in stru in-place to match the ordered_stru's ordering.
+    Args:
+        stru: 
+        ordered_stru: the Structure object that has the desired order.
+    Returns:
+        Nothing.
+    """
+    for res, ordered_res in zip(stru.residues, ordered_stru.residues):
+        if res.name == ordered_res.name:
+            res.atoms = ordered_res.atoms
```

Comment:
This is problematic.

Reply:
This function should go to `structure_operation/general.py`. Sorry for the misleading direction before.

Reply:
It will be great to rename ordered_stru as ref_stru and change the name of the function more specific. By specific, I mean this function cannot handle cases like stru and ref_stru dont have the same order/number of residues. Such limitation should be reflected on the function name.

Reply:
For the new algorithm, I assume that you forgot to delete `res.atoms = ref_res.atoms`
But still, there could be a better way of reordering a list. We can discuss it in our meeting tomorrow.